bokeh/io/saving.py

```diff
@@ -94,7 +103,8 @@ def save(obj, filename=None, resources=None, title=None, template=None, state=No
 # Private API
 #-----------------------------------------------------------------------------
 
-def _get_save_args(state, filename, resources, title):
+def _get_save_args(state: State, filename: Optional[str], resources: Optional[Resources],
+        title: Optional[str]) -> Tuple[str, Resources, str]:
```

Comment:
Is this how we generally handle hanging args in the codebase?  I generally find this much more readable and that goes double once type annotations are added
```python
def _get_save_args(state: State, 
                   filename: Optional[str], 
                   resources: Optional[Resources],
                   title: Optional[str]) -> Tuple[str, Resources, str]:
```

Reply:
I mentioned in other threads, I hate this style, i.e. having this huge gap in front of the code and having so much variability in alignment of blocks. The style I'm using here is the natural extension of the original, without delving into stylistic considerations. This can be always refactored when we settle on a style.

Reply:
Just noting from the other thread that for lines that require splitting, I think we agree on
```python
def _get_save_args(
    state: State, 
    filename: Optional[str], 
    resources: Optional[Resources],
    title: Optional[str]
) -> Tuple[str, Resources, str]:
```